### DISCUSSION TITLE
Designing the timeline feature

### DISCUSSION THREAD

**Discussion by dbaslan** [SWE574-Fall2023-Group1/SWE574-Fall2023-G1#114] [Ideas]
@SWE574-Fall2023-Group1/all Folks we need to rethink our timeline for both web and mobile. We got feedback from the professor today (somebody please share more detailed notes) but basically she does NOT want a plain sorted list. We need to make the images more prominent, render an actual visual line with chronological ordering, let the user specify a date range, and figure out how to choose a small set of memories from the larger list of relevant posts. She mentioned features such as sliding/dragging through the timeline and a play button to move automatically. We could prepare a proposal to see if our design will be accepted. Let's write down our ideas and we can discuss them in the next meeting. I don't think we can get away with a quick and dirty solution for this feature.

**Comment by sadikkuzu** [comment]
@dbaslan hi,
What is the latest situation of this frontend feature design? I've seen mockups and web implementations in wiki&repo.
Could you briefly inform us about what has done, what is going on and what will be done on later weeks?
Thanks in advance.
